packages/components/CHANGELOG.md

```diff
@@ -2,9 +2,19 @@
 
 ## Unreleased
 
+### New Feature
+
+-   Add `none` as one of the possible values for the `size` prop in `Card`, `CardBody`, `CardHeader`, and `CardFooter` ([#32566](https://github.com/WordPress/gutenberg/pull/32566)).
```

Comment:
`size: none` looks a bit awkward in a README. I've also commented about that later in the review.